noir-projects/noir-contracts/contracts/avm_test_contract/src/main.nr

```diff
@@ -144,6 +144,14 @@ contract AvmTest {
         added
     }
 
+    #[aztec(public)]
+    fn variable_base_msm() -> [Field; 3] {
+        let g = EmbeddedCurvePoint { x: 1, y: 17631683881184975370165255887551781615748388533673675138860, is_infinite: false };
+        let scalar = EmbeddedCurveScalar { lo: 3, hi: 0 };
+        let triple_g = multi_scalar_mul([g], [scalar]);
```

Comment:
Maybe good to test other lengths as well (many elements in the list). For transpilation alignment sake.